GOVERNANCE.md

```diff
@@ -172,36 +235,46 @@ Each voter can vote zero or more times, once per each candidate. As this is not
 * If a whole-council vote succeeds, the council is dissolved and a new council election is triggered immediately.
 
 #### Ejecting Core Contributors
-* Core contributors can be ejected through a simple majority vote by the council
+* Core contributors can be ejected through a simple majority vote by the council. Council members vote "Yes" or "No".
 * Upon ejecting a core contributor the council must publish an issue ticket, or public document detailing the
   * Violations
   * Evidence if available
   * Remediation plan (if necessary)
   * Signatures majority of council members to validate correctness and accuracy
 * Core contributors can also voluntarily leave the project by notifying the community through a public means or by notifying the entire council.
 
-### One Time Decisions
-* The first election will also include a vote for the number of Council Members, options being 5 and 7. A simple majority will indicate preference.
-  * At the conclusion of the vote this section should be removed
-
 #### Voting Criteria For Future Elections
 Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
-* For the first election only the people registered following the guidelines in elections/ArviZ_2020.md can vote
+* For the first election only the people registered following the guidelines in `elections/ArviZ_2020.md` can vote
 * In the first year, the council must determine voting eligibility for future elections between two criteria:
   * Core contributors
   * The contributing community at large
 
 ### Core Contributors
 Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
 These can be, but are not limited to, code contributions, community contributions, tutorial
-development etc.
+development etc. Core Contributors will be given the ability to manage the ArviZ GitHub
+repository, including code merges to master. This does not necessarily mean Core Contributors
+must submit code, but more so signifies trust with the project as a whole.
+
+#### Core Contributor Responsibilities
+* Enforce code of conduct
+* Maintain a check against Council
```

Comment:
How do they function as a check against the Council?

Reply:
I understand that the idea is that the check is generally informal with the vote of no confidence as last resort if discussion and warnings in github and slack do not work